package.json

```diff
@@ -19,6 +19,7 @@
     "test:codacy-upload": "cat ./coverage-report/lcov.info | codacy-coverage",
     "test:codecov-upload": "codecov -f ./coverage-report/lcov.info",
     "test:smoke": "./bin/run-smoke-tests.sh",
+    "test:integration": "ts-node --require tsconfig-paths/register ./node_modules/.bin/codeceptjs run --config default.conf.js --reporter mocha-multi",
```

Comment:
no docker here?

Reply:
It's the entry point for codecept, just like `test` task used to be in the old repo.